src/py/_pyodide/_importhook.py

```diff
@@ -135,7 +135,15 @@ def register_js_finder() -> None:
 
 STDLIBS = sys.stdlib_module_names | {"test"}
 # TODO: Move this list to js side
-UNVENDORED_STDLIBS = ["distutils", "ssl", "lzma", "sqlite3", "hashlib"]
+UNVENDORED_STDLIBS = [
+    "distutils",
+    "ssl",
+    "lzma",
+    "sqlite3",
+    "hashlib",
+    "pydoc_data",
+    "pydecimal",
+]
```

Comment:
In the future, maybe we could switch to putting this information in `repodata.json`.

Reply:
Indeed, this should not be hardcoded inside pyodide-py, and putting this inside repodata.json would be nice.